### PR TITLE
buff the illegal rollingpin

### PR DESCRIPTION
**Pull request by TheBoondock**
## About The Pull Request
chef has an illegal rollingpin in his vendor but its the same as the regular, so i simply made a subtype with higher force (subject to change)
New sprite just dropped, courtesy of @Wallemations 
metalpin with handle ss13

## Why It's Good For The Game
image

## Changelog

TheBoondock, Wallemation 
:cl:
balance: the contraband rollingpin in chef's vendor is now made of metal and yes it hurts alot more when being hit
/:cl:

### CODE REVIEW

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -114,6 +114,9 @@
 	custom_price = PAYCHECK_CREW * 1.5
 	tool_behaviour = TOOL_ROLLINGPIN
 
+/obj/item/kitchen/rollingpin/illegal
+	force = 20
```
Comment (JohnFulpWillard) [inline code/game/objects/items/kitchen.dm:118]:
```suggestion
	force = 60
```

Comment (san7890) [reply code/game/objects/items/kitchen.dm:118]: .

**Review by MrMelbert — CHANGES_REQUESTED** [review]
if you're serious make it 12 or 15 force, 10 wound bonus, and give it a unique name at least (i'd like a sprite like a metal rolling pin but w/e)

**Comment by ArcaneMusic** [comment]
> if you're serious make it 12 or 15 force, 10 wound bonus, and give it a unique name at least (i'd like a sprite like a metal rolling pin but w/e)

The Roller Pin? (Like it's an industrial conveyor belt roller)

**Comment by Shadowflame909** [comment]
Sprite edit and change it from tan to black. Now its a syndicate rolling pin

**Comment by ATH1909** [comment]
needs a different name to distinguish it from a normal one

**Comment by newfren** [comment]
20 is so fucking high lmao

**Comment by wesoda25** [comment]
No no it's good, this will introduce a new rolling-pin centric meta to the game

**Comment by optimumtact** [comment]
if this goes stale again ur dead melb

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/game/objects/items/kitchen.dm [hunk 2]
```diff
@@ -114,6 +114,15 @@
 	custom_price = PAYCHECK_CREW * 1.5
 	tool_behaviour = TOOL_ROLLINGPIN
 
+/obj/item/kitchen/rollingpin/illegal
+	name = "illegal rolling pin"
+	desc = "A heavy metalic rolling pin to bash in those annoying ingredients"
+	icon_state = "metal_rolling_pin"
+	force = 12
```
[the remark this answers is not in the document]
Comment (TheBoondock) [reply code/game/objects/items/kitchen.dm:121]: Idk bro the handle kinda insulated it no?

**Review by MrMelbert — APPROVED** [review]

**Pull request merged**